src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -388,6 +389,22 @@ public static String getPathWithoutBucket(String path) {
         return CloudStorageFileSystem.forBucket(BUCKET).getPath(pathWithoutBucket);
     }
 
+    /**
+     * String -> Path. This will accept either
+     * "folder/file", or
+     * "file://folder/file", or
+     * "gs://bucket/folder/file".
+     * It will only accept "hdfs://" if you have an NIO filesystem provider installed for HDFS.
+     */
+    public static java.nio.file.Path getPath(String fileNameOrUrl) {
```

Comment:
There is a similar method in IOUtils#getPath that should be used instead of adding another one here. It will need to have the GCS special casing added though.